File: beginner/javascript/1175.js
var input = require('fs').readFileSync('stdin', 'utf8');
var linhas = input.split('\n');

var i, temp;
var n = [];

for (i = 0; i < 20; i++) {
    n[i] = parseInt(linhas.shift());
}

for (i = 0; i < 10; i++) {
    temp = n[i];
    n[i] = n[19 - i];
    n[19 - i] = temp;
}

for (i = 0; i < 20; i++) {
    console.log(`N[${i}] = ${n[i]}`);
}

File: beginner/python/1175.py
n = []

for i in range(0,20):
    valor = int(input())
    n.append(valor)


for i in range(0,10):
    temp = n[i]
    n[i] = n[19 - i]
    n[19 - i] = temp


for i, value in enumerate(n):
    print("N[%d] = %d" % (i, value))
